src/deepsparse/pipeline.py

```diff
@@ -247,7 +248,12 @@ def __call__(self, *args, **kwargs) -> BaseModel:
             )
 
             # submit split batches to engine threadpool
-            batch_outputs = list(self.executor.map(self.engine_forward, batches))
+            engine_forward_with_context = partial(
+                self.engine_forward, context=postprocess_kwargs
```

Comment:
thinking we should rename `postprocess_kwargs` to `context` here since it's used for more than postprocess

Reply:
cool i agree, addressed this in the latest commit!